docker-alfresco/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM alfresco/alfresco-base-tomcat:8.5.43-java-11-openjdk-centos-7
 # Set default user information
 ARG GROUPNAME=Alfresco
 ARG GROUPID=1000
-ARG USERNAME=alfresco
+ARG IMAGEUSERNAME=alfresco
```

Comment:
Should we bring this change in on another PR? This fixes an error cause by the fabric 8 plugin when building in Windows. The variable USERNAME cannot be assigned by the dockerfile and the machine username is instead used. As such the image doesn't have the user "alfresco" setup correctly and errors can be thrown when amps are applied.

Reply:
Removed and separate PR created (#657)